### PR TITLE
Add script to generate runtime dependencies for each project - first draft

### PR DESCRIPTION
**Pull request by IgorTodorovskiIBM**

### CODE REVIEW

File: tools/generate_runtime_dependencies.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Python script to determine the runtime dependencies of open source tools hosted on Github
+# The dependencies are stored in a dictionary and printed in a markdown table format
+# Only unique dependencies are stored and printed, to avoid duplication
+# Note: Only run this on z/OS
```
Comment (MikeFultonDev) [inline tools/generate_runtime_dependencies.py:4]: Can you add a comment that says what the input and output is and what env vars are required?

**Review by MikeFultonDev — CHANGES_REQUESTED** [review]
other than my one comment, rest looks good

**Comment by IgorTodorovskiIBM** [comment]
This PR needs a bit of rework, will work on it this week

**Comment by IgorTodorovskiIBM** [comment]
Will revisit this at a later point or if requested

**Pull request closed without merging**